src/utils/craftLayerURL.js

```diff
@@ -10,8 +12,15 @@ export const createLayerURL = (location, type) => {
     pathname,
     search,
   } = location;
+  const splittedPath = pathname.split('/');
 
-  const url = `${pathname}${search}`;
+  if (type === LAYER_TYPES.CREATE) {
+    splittedPath.length = 4;
```

Comment:
Maybe call this `pathList` instead? `splitted` isn't a recognized English word :)